src/metrics/processes.js

```diff
@@ -0,0 +1,46 @@
+'use strict'
+
+const exec = require('../utils/exec.js')
+const os = require('os')
+
+module.exports = class ProcessesMetric {
+  constructor (io, conf) {
+    this.io = io
+    this.conf = conf
+    this.processCount = io.metric({
+      name: 'Total Processes',
+      type: 'os/process/alive'
+    })
+    this.zombieProcessCount = io.metric({
+      name: 'Zombie processes',
+      type: 'os/process/zombie'
+    })
+
+    this.refreshInterval = parseInt(conf.disk_refresh_rate) || 60
+
+    // use different command for macos
+    const fetchImpl = os.platform() === 'darwin' ? this.fetchDarwin.bind(this) : this.fetch.bind(this)
+    this._worker = setInterval(fetchImpl, this.refreshInterval * 1000)
+    fetchImpl()
+  }
+
+  fetchDarwin () {
+    exec('ps -A', (err, stdout) => {
+      if (err || stdout.length === 0) return console.error(`Failed to retrieve process count for darwin`, err)
+      this.processCount.set(stdout.split('\n').length - 1)
+    })
+  }
+
+  fetch () {
```

Comment:
On win, doesn't crash but make logs (same for all unix commands)

Reply:
Expected, i think its okay to log errors